java/ql/src/Security/CWE/CWE-113/NettyResponseSplitting.ql

```diff
@@ -12,6 +12,8 @@
  *       external/cwe/cwe-093
  *       external/cwe/cwe-113
  */
+overlay[local?]
+module;
```

Comment:
I'm uneasy about using file-level overlay annotations in a `.ql` file. Somehow, I actually thought we did not allow it.
The reason I'm uneasy about it is that the select clause itself is in the scope of that annotation, but does not get compiled as local currently. However, we might want to change that eventually, for example to support updgrade scripts.

Could you annotated some of the individual classes instead?

Reply:
That is a good point. I've changed it to annotate individual classes instead. In this case it is everything in the file except the select. 

> However, we might want to change that eventually, for example to support upgrade scripts.

Then we should add a warning / error when file-level overlay annotations are used in `.ql` files as quickly as possible.